src/webhooks/index.ts

```diff
@@ -66,6 +67,40 @@ export default function (
             (event.data.object as Stripe.PaymentMethod).id,
           );
           break;
+
+        case 'setup_intent.succeeded': {
+          stripe.subscriptions
```

Comment:
`await` this

Reply:
I didn't add `await` here because this promise has `then` and `catch`.